src/Microsoft.Crank.Agent/Microsoft.Crank.Agent.csproj

```diff
@@ -29,6 +29,11 @@
     <PackageReference Include="NuGet.Versioning" Version="5.5.1" />
     <PackageReference Include="Vanara.PInvoke.Kernel32" Version="3.3.14" />
     
+    <PackageReference Include="Serilog.Extensions.Logging" Version="3.0.1" />
```

Comment:
That's a lot of direct references just for a file logger. Are they all required?

Reply:
You are right, Serilog.AspNetCore includes Serilog.Extensions.Logging, so not needed